tests/test_indexer.py

```diff
@@ -330,6 +294,70 @@ def get_search_results(cls, query, expected_hit_count):
             else:
                 time.sleep(0.5)
 
+    def create_sample_bundle_created_event(self, bundle_key):
+        raise NotImplemented()
+
+    def process_new_indexable_object(self, event, logger):
+        raise NotImplemented()
+
+    @classmethod
+    def setUpClass(cls):
+        Config.set_config(DeploymentStage.TEST_FIXTURE)
+        cls.blobstore, _, cls.test_fixture_bucket = Config.get_cloud_specific_handles(cls.replica)
+        Config.set_config(DeploymentStage.TEST)
+        _, _, cls.test_bucket = Config.get_cloud_specific_handles(cls.replica)
+        cls.es_server = ElasticsearchServer()
```

Comment:
ElasticsearchServer startup/shutdown could be moved to setUpModule/tearDownModule. This may entail making making es_server a global within this file, yet the improvement in test execution time by eliminating an unnecessary startup/shutdown seems worth it.

Reply:
moved to module setup/teardown level